server/main.go

```diff
@@ -24,15 +24,29 @@ import (
 	"github.com/tigrisdata/tigris/store/search"
 	"github.com/tigrisdata/tigris/util"
 	ulog "github.com/tigrisdata/tigris/util/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
+func initTracer() {
+	if !config.DefaultConfig.Tracing.Enabled {
+		return
+	}
+	tracer.Start(
+		tracer.WithUDS("/var/run/datadog/apm.socket"),
```

Comment:
should we move this to config as well?

Reply:
I was contemplating about adding further options but decided to delay adding more code until I validate this works as expected. Giving it a second though it is very little work so you're right, might as well add it!